modules/iam__enum_roles/main.py

```diff
@@ -48,32 +67,44 @@ def main(args, pacu_main):
     with open(word_list_path, 'r') as f:
         word_list = f.read().splitlines()
 
-    print('Warning: This script does not check if the keys you supplied have the correct permissions. Make sure they are allowed to use iam:UpdateAssumeRolePolicy on the role that you pass into --role-name and are allowed to use sts:AssumeRole to try and assume any enumerated roles!\n')
+    print(
+        'Warning: This script does not check if the keys you supplied have the correct permissions. Make sure they are '
+        'allowed to use iam:UpdateAssumeRolePolicy on the role that you pass into --role-name and are allowed to use '
+        'sts:AssumeRole to try and assume any enumerated roles!\n'
+    )
 
     data = {
         'attempts': 0,
         'valid_roles': [],
         'roles_assumed': []
     }
 
-    client = pacu_main.get_boto3_client('iam')
-
     print('Targeting account ID: {}\n'.format(args.account_id))
     print('Starting role enumeration...\n')
-
     for word in word_list:
+        sys.stderr.write('.')
         role_arn = 'arn:aws:iam::{}:role/{}'.format(args.account_id, word)
 
         data['attempts'] += 1
 
         try:
-            client.update_assume_role_policy(
-                RoleName=args.role_name,
-                PolicyDocument='{{"Version":"2012-10-17","Statement":[{{"Effect":"Deny","Principal":{{"AWS":"{}"}},"Action":"sts:AssumeRole"}}]}}'.format(role_arn)
+            iam.update_assume_role_policy(
+                RoleName=role_name,
+                PolicyDocument='''
+                {{
+                    "Version":"2012-10-17",
+                    "Statement":[{{
+                        "Effect":"Deny",
+                        "Principal":{{"AWS":"{}"}},
+                        "Action":"sts:AssumeRole"
+                    }}]
+                }}'''.format(role_arn).strip()
```

Comment:
Double brackets are needed here to escape interpolation with the format function.